samcli/lib/build/build_graph.py

```diff
@@ -25,38 +30,77 @@ def __init__(self, msg):
         Exception.__init__(self, msg)
 
 
-def _build_definition_to_toml_table(build_definition):
+def _function_build_definition_to_toml_table(function_build_definition):
     """
-    Converts given build_definition into toml table representation
+    Converts given function_build_definition into toml table representation
 
-    :param build_definition: BuildDefinition
-    :return: toml table of BuildDefinition
+    :param function_build_definition: FunctionBuildDefinition
+    :return: toml table of FunctionBuildDefinition
     """
     toml_table = tomlkit.table()
-    toml_table[CODE_URI_FIELD] = build_definition.codeuri
-    toml_table[RUNTIME_FIELD] = build_definition.runtime
+    toml_table[CODE_URI_FIELD] = function_build_definition.codeuri
+    toml_table[RUNTIME_FIELD] = function_build_definition.runtime
+    toml_table[SOURCE_MD5_FIELD] = function_build_definition.source_md5
     toml_table[FUNCTIONS_FIELD] = \
-        list(map(lambda f: f.name, build_definition.functions))
+        list(map(lambda f: f.functionname, function_build_definition.functions))
 
-    if build_definition.metadata:
-        toml_table[METADATA_FIELD] = build_definition.metadata
+    if function_build_definition.metadata:
+        toml_table[METADATA_FIELD] = function_build_definition.metadata
 
     return toml_table
 
 
-def _toml_table_to_build_definition(uuid, toml_table):
+def _toml_table_to_function_build_definition(uuid, toml_table):
     """
-    Converts given toml table into BuildDefinition instance
+    Converts given toml table into FunctionBuildDefinition instance
+
+    :param uuid: key of the function toml_table instance
```

Comment:
Please use [numpy syntax](https://github.com/aws/aws-sam-cli/blob/346ff2923477d8bc0f4fffb5da3629bcfc935b96/DEVELOPMENT_GUIDE.md#code-conventions) for docstrings.